README.md

```diff
@@ -6,7 +6,7 @@ Shopify Application Rails engine and generator
 
 Description
 -----------
-This gem includes a Rails Engine and generators for writing Rails applications using the Shopify API. The Engine provides a SessionsController and all the required code for authenticating with a shop.
+This gem includes a Rails Engine and generators for writing Rails applications using the Shopify API. The Engine provides a SessionsController and all the required code for authenticating with a shop via the Partner (Oauth) style of authentication. Private App's token based auth is not supported.
 
```

Comment:
Can we just write `with a shop via Oauth (other authentication methods are not supported).`

Its shorter and it covers some of the deprecated auth methods as well.